packages/cli/src/hl7.ts

```diff
@@ -0,0 +1,59 @@
+import { formatHl7DateTime, Hl7Message } from '@medplum/core';
+import { Hl7Client, Hl7Server } from '@medplum/hl7';
+import { Command } from 'commander';
+import { readFileSync } from 'fs';
+import { createMedplumCommand } from './util/command';
+
+const send = createMedplumCommand('send')
+  .description('Send an HL7 v2 message via MLLP')
+  .argument('<host>', 'The destination host name or IP address')
+  .argument('<port>', 'The destination port number')
+  .argument('[body]', 'Optional HL7 message body')
+  .option('--generate', 'Generate a sample HL7 message')
```

Comment:
We may want to consider renaming this to e.g. `example` to more clearly express what it does